STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/src/_az_ulib_dm_blob.c

```diff
@@ -175,118 +142,79 @@ static az_result result_from_hal_status(HAL_StatusTypeDef status)
   }
 }
 
-static az_result copy_blob_to_flash(NXD_ADDRESS* ip, CHAR* resource, CHAR* host, void* address)
+static uint8_t local_buffer[256] = { 0 }; // define in BSS to reduce load on stack
```

Comment:
>local_buffer [](http://example.com/codeflow?start=15&length=12)

You can reduce the scope of this variable keeping it static but inside of the function.